luxonis_train/nodes/heads/discsubnet_head/discsubnet_head.py

```diff
@@ -0,0 +1,67 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+from luxonis_train.enums import TaskType
+from luxonis_train.nodes.base_node import BaseNode
+from luxonis_train.utils import (
+    Packet,
+)
+
+from .blocks import Decoder, Encoder
+
+
+class DiscSubNetHead(BaseNode[Tensor, Tensor]):
+    in_channels: list[int] | int
+    out_channels: list[int] | int
+    base_channels: int
+    out_features: bool
+    tasks: list[TaskType] = [TaskType.SEGMENTATION]
+
+    def __init__(
+        self,
+        in_channels: list[int] | int = 6,
+        out_channels: list[int] | int = 2,
+        base_channels: int = 64,
+        out_features: bool = False,
+        **kwargs,
+    ):
```

Comment:
Same as with `RecSubNet` let's include docstrings and short explanation with credits